src/common/filters/http-exception.filter.ts

```
@@ -0,0 +1,69 @@
+import {
+  ExceptionFilter,
+  Catch,
+  ArgumentsHost,
+  HttpStatus,
+  UnauthorizedException,
+} from '@nestjs/common';
+import { Request, Response } from 'express';
+import axios from 'axios';
+import { BaseException } from '../exception/base.exception';
+import { ErrorResponseDto } from '../dto/response/error-response.dto';
+import { getAppMode, getAppModeEnv } from '../utility';
```

ESLint

[error] 'getAppModeEnv' is defined but never used.

(@typescript-eslint/no-unused-vars)